### DISCUSSION TITLE
Enabling GitHub Pilot access to GitHub Projects.

### DISCUSSION THREAD

**Discussion by kmoa** [community/community#145290] [Enterprise]
### Select Topic Area

Question

### Body

Would it be possible to add GitHub Projects as a source of information for the pilot, enabling the user to ask about the issues that are tracked in the Project?

![image](https://github.com/user-attachments/assets/d8f83e45-fbfd-4ed3-af5c-cf421ca26872)

It's can access it directly, but are there ways around it?

**Comment by Akash1134** [comment]
Hi @kmoa 

Currently, GitHub Copilot does not have direct access to GitHub Projects to fetch information about issues that are tracked there. However, you can manually query issues within a repository using GitHub's search and API functionalities to gather similar information.

Thanks!
